src/gui/qgssourceselectproviderregistry.h

```diff
@@ -89,6 +90,22 @@ class GUI_EXPORT QgsSourceSelectProviderRegistry
       QgsProviderRegistry::WidgetMode widgetMode
     );
 
+  signals:
+
+    /**
+     * Emitted whenever a provider is added to the registry.
+     *
+     * \since QGIS 3.30
```

Comment:
Should this be edited out?

Reply:
I'm not sure -- I'm personally inclined to just let things like this go in order to ease the pain of backports, and because it's erring on the "safe" side (ie at worst someone won't use the method when they potentially could, as opposed to using it when they can't)

Reply:
@nyalldawson , fair enough